crates/sui-types/src/transaction_displays.rs

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::transaction::CallArg::Pure;
+use crate::transaction::{
+    write_sep, Argument, CallArg, Command, ObjectArg, ProgrammableMoveCall, ProgrammableTransaction,
+};
+use std::fmt::{Display, Formatter};
+use tabled::{
+    builder::Builder as TableBuilder,
+    settings::{style::HorizontalLine, Panel as TablePanel, Style as TableStyle},
+};
+
```

Comment:
Given that this is a new file, ideally, we should add some comments to explain what's the purpose of this file and when to use these stuff.